src/heitech.zer0mqXt.core/Transport/RequestReplyMessage.cs

```diff
@@ -15,6 +15,17 @@ internal RequestReplyMessage(SocketConfiguration configuration, TMessage message
             _isSuccess = success;
         }
 
+        private RequestReplyMessage(SocketConfiguration configuration, string message)
```

Comment:
To me this is a bit inconsistent:
- the new _c'tor_ is `private` with a factory method
- but the already existing _c'tor_ is `internal` without a factory method

On the other hand, I believe this type is only used internally, so there is no user which could get potentially confused.
Perhaps this entiry type should be `internal`?

Reply:
Nevermind, I just noticed #28!